src/swagger/specGenerator.ts

```diff
@@ -100,7 +100,11 @@ export class SpecGenerator {
     const pathMethod: Swagger.Operation = pathObject[method.method] = this.buildOperation(controllerName, method);
     pathMethod.description = method.description;
     pathMethod.summary = method.summary;
-    pathMethod.tags = method.tags;
+    
+    pathMethod.tags = [controllerName].concat(method.tags || []) 
+    
+    // Use operationId tag otherwise fallback to generated. Warning: This doesn't check uniqueness.
```

Comment:
Should uniqueness be checked?

Reply:
Okay, let's go without it for now. Then if people later want it, we can still add it in.